src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -653,4 +666,59 @@ public function installDrupal() {
     ]);
   }
 
+  /**
+   *
+   * Truncates the history from the deployment branch on the remote repo.
+   * This reduces the number of objects in git and drastically reduces
+   * deployment times on much larger projects.
+   * Trigger with --truncate-history option
```

Comment:
Will that work with truncate-history being configured in yml file of blt?

Reply:
I've added a `deploy.truncate_history` config value so this can now be configured in the blt yml file

Reply:
Perfect